webapp/src/DOMJudgeBundle/Resources/views/partials/scoreboard_table.html.twig

```diff
@@ -302,7 +304,7 @@
 
                             <i class="fas fa-clock fa-fw"></i>
                             <span style="font-size:90%;" title="first solved">
-                                {% if scoreboard.summary.problem(problem.probid).bestTimeInMinutes(0) is not null %}
+                                {% if problem.probid in firsttosolveknown %}
```

Comment:
We still use the non-Symfony scoreboard for the public and team pages (as these parts are not in Symfony yet at all). Shouldn't we change the logic there too?

Reply:
Unfortunately, yes.

Reply:
Fortunately, no.